lib/actions/deleteObject.js

```diff
@@ -1,31 +1,94 @@
 const { messages } = require('elasticio-node');
 const MetaLoader = require('../helpers/metaLoader');
 const sfConnection = require('../helpers/sfConnection.js');
+const helpers  = require('../helpers/deleteObjectHelpers.js')
 
+/**
+ * Des: Taken from lookupObject.js due to overlapping functionality
+*/
+module.exports.objectTypes = function objectTypes(configuration) {
+  const metaLoader = new MetaLoader(configuration, this);
+  return metaLoader.getSearchableObjectTypes();
+};
 
-exports.objectTypes = function objectTypes(configuration) {
+/**
+ * Des: Taken from lookupObject.js due to overlapping functionality
+*/
+module.exports.getMetaModel = async function getMetaModel(configuration) {
+  configuration.metaType = 'lookup';
   const metaLoader = new MetaLoader(configuration, this);
-  return metaLoader.getObjectTypes();
+  const metaData = await metaLoader.loadMetadata();
+  metaData.in.properties[configuration.lookupField].required = true;
+  return metaData;
 };
 
+/**
+ * Des: 
+ * - This function will firstly lookup the object based on fields
+ * in the metaData
+ * - Emits an error if zero or more than one object is found and returns false
+ * - If delete was sucessful
+ * @implementedBy C.J.V
+ * @param configuration encapsulates the lookup value and lookup key
+*/
+module.exports.process = async function process(message, configuration) {
+  const { lookupField } = configuration;
```

Comment:
I cant see this field in component.json file.

Reply:
Before creating PR please make sure everything is works on platform. To do this:
1. Push component to platform
2. Manually test it on some simple cases

Reply:
I apologize not all changed files were present. In this push, I've confirmed the component operates as expected for simple test cases in an e.io environment